### DISCUSSION TITLE
Production deployment failing on vercel returns 500 error code when trying to GET /

### DISCUSSION THREAD

**Discussion by Nessvah** [withastro/docs#4595] [Ideas]
Hi guys. I was working on a small portfolio using astro with react and tailwind integrations, I already had it deployed but when I added more content to the portfolio, my next deployment failed. 

I tried to fix it but I wasn't able to solve it yet. For what it seems, checkly makes a get to get the page "/" but then it gets no answer from the server and must get a timeout with the error 500.

My build is working locally and doesn't give me any error of any kind. 

Can you help me?? Any suggestions?

This is the error that appears when visiting the website: 

This Serverless Function has crashed.

Your connection is working correctly.

Vercel is working correctly.

500: INTERNAL_SERVER_ERROR
Code: FUNCTION_INVOCATION_FAILED
ID: cdg1::gq8cd-1694182000076-250bd676e51a

This is the log that I got from checkly:

Sep 08 14:44:54 - DEBUG - Starting job
Sep 08 14:44:54 - DEBUG - Creating runtime using version 2023.02
Sep 08 14:44:54 - DEBUG - Running Playwright test script
Sep 08 14:44:56 - INFO  - Running 1 test using 1 worker
Sep 08 14:44:57 - INFO  - [1/1] [chromium] › test.spec.js › visit page and take screenshot
Sep 08 14:44:58 - INFO  - 1) [chromium] › ../../checkly/functions/src/2023-02/node_modules/vm2/lib/bridge.js:485:11 › visit page and take screenshot 

    Error: should respond with correct status code

    Expected: < 400
    Received:   500

/tmp/19g67lmanfybm/script.js:16:11

    attachment #1: video (video/webm) ──────────────────────────────────────────────────────────────
    artifacts/videos/visit-page-and-take-screenshot-1694180698481.webm
    ────────────────────────────────────────────────────────────────────────────────────────────────

    attachment #2: trace (application/zip) ─────────────────────────────────────────────────────────
    artifacts/script-visit-page-and-take-screenshot-chromium/trace.zip
    Usage:

        npx playwright show-trace artifacts/script-visit-page-and-take-screenshot-chromium/trace.zip

    ────────────────────────────────────────────────────────────────────────────────────────────────
Sep 08 14:44:58 - INFO  - 1 failed
    [chromium] › ../../checkly/functions/src/2023-02/node_modules/vm2/lib/bridge.js:485:11 › visit page and take screenshot
Sep 08 14:44:58 - DEBUG - Run finished
Sep 08 14:44:58 - DEBUG - Uploading log file

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

I guess you fixed your issue by now. And, the discussions are intended to discuss improvements to the documentation; for any support questions, feel free to join our [Discord server](https://astro.build/chat)! So, I'm closing this.

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
